README.md

```diff
@@ -1,2 +1,90 @@
 # services
 Overview of all OpenML components including a docker-compose to run OpenML services locally
+
+## Overview
+
+![OpenML Component overview](./documentation/OpenML-overview.png)
+
+
+
+
+## Prerequisites
+- Linux/MacOS/Windows (should all work)
+- [Docker](https://docs.docker.com/get-docker/) 
+- [Docker Compose](https://docs.docker.com/compose/install/) version 2.21.0 or higher
+
+## Usage
+
+You run all OpenML services locally using
+```bash
+docker compose --profile all up -d
+```
+Stop it again using 
+```bash
+docker compose --profile all down
+```
+
+
+You can use different profiles, for example:
+```bash
+docker compose --profile all up -d    # all services
+docker compose up -d                  # only the database
+docker compose --profile frontend -d  # Frontend, rest-api, elasticsearch and database
+```
+Use the same profile for your `down` command.
```

Comment:
An overview of the profiles and which services they spin up would be nice.

Reply:
Good idea! Done https://github.com/openml/services/pull/3/commits/2f1201fd61e9e8259c41e37ef5ecc30787ca3970